src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -393,15 +395,24 @@ describe( 'CKEditorComponent', () => {
 						const editable = component.instance.editable();
 						editable.fire( 'paste', pasteEventMock );
 
-						return whenEvent( 'paste', component ).then( () => {
-							expect( spy ).toHaveBeenCalledTimes( 1 );
-							expect( component.instance.getData() ).toEqual( '<p>bam</p>\n' );
-						} );
+						if ( !( CKEDITOR.env.ie && CKEDITOR.env.version <= 11 ) ) {
+							return whenEvent( 'paste', component ).then( () => {
+								expect( spy ).toHaveBeenCalledTimes( 1 );
+								expect( component.instance.getData() ).toEqual( '<p>bam</p>\n' );
+							} );
+						} else {
+							// Test if IE emits paste event
+							spyOn( component.paste, 'emit' );
```

Comment:
The easiest way to check if the test covers what intended is to modify the code it should test. In this case it would be commenting out `paste` event emitting: https://github.com/ckeditor/ckeditor4-angular/blob/910bcaa07587d1cb06275f8c2e6d00efce0b5aba/src/ckeditor/ckeditor.component.ts#L341-L345

Unfortunately, the test still passes and the reason is simple - first you spy on `spyOn( component.paste, 'emit' );` and in the next line you just simply call this method with `component.paste.emit();` which triggers the spy. So it doesn't need any external code to pass.